test/integration/role.js

```diff
@@ -0,0 +1,62 @@
+/* global expect, chai, agent */
+
+const helpers = require('./helpers');
+
+/*
+ * The /roles API endpoint
+ *
+ * This test suite implements full CRUD on the /projects HTTP API endpoint.
+ */
+describe('(/roles) The roles API endpoint', () => {
+  // project we will add during this test suite.
+  const roles = {
+    label : 'Receptionniste',
+    project_id : 1,
+  };
+  const adminUuid = '5b7dd0d6-9273-4955-a703-126fbd504b61';
+  const roleAdmin = {
+    label : 'Administrator',
+  };
+  const userRole = {
+    user_id : 2,
+    role_uuids : adminUuid,
+  };
+
+  it('GET /roles returns a list of roles', () => {
+    return agent.get('/roles?project_id='.concat(roles.project_id))
```

Comment:
Just FYI - you can use chai-http's `.query()` method to pass queries as an object.  Like this:
```js
return agent.get('/roles')
  .query({ project_id : 1 });
 // this becomes HTTP GET /roles?project_id=1
```
It makes it easier to update and change values, since you can then do `agen.get('/roles').query(roles);` for example.